packages/asset-swapper/src/types.ts

```diff
@@ -253,6 +253,7 @@ export interface SwapQuoterOpts extends OrderPrunerOpts {
         warningLogger?: LogFunction;
         infoLogger?: LogFunction;
     };
+    samplerCallDataOverrides?: Partial<CallData>;
```

Comment:
Why is this a `CallData` type and not a `GethCallOverrides` from `ethereum-types`? `CallData` seems like overspecification since we don't care about the `from` field?